### ISSUE TITLE
How turn on Centos Server

### ISSUE THREAD

**Issue by cfreire10** [basildane/WakeOnLAN#204]
Hi friends, I hope that you be fine. I would like to know if to turn on my Centos Server is just necessary to set its IP , mask and mac address to achieve it, or is it necessary another setting from my w10 pc ? Thanks for your attention.

**Comment by basildane** [comment]
All it needs is a MAC address and connectivity to the network.
Your server must be able to handle WOL (bios configured).

**Comment by cfreire10** [comment]
Yes i have all the requeriment, but I et the next error, in attached image. Thanks.
![WhatsApp Image 2022-12-14 at 6 25 13 PM](https://user-images.githubusercontent.com/120610976/208305676-77d83f90-d87f-4b73-ad25-da47328d31dc.jpeg)

**Comment by Moopere** [comment]
Looks like you're trying to shutdown the server but the server is reporting no RPC.  This sounds about right given its not a windows server - I've not done this myself but imagine you'll need to use the 'custom' shutdown method and enter credentials that will work for your Centos server.

![image](https://user-images.githubusercontent.com/17094611/221392222-05e9fd3b-3547-426c-881f-df06cff40cbc.png)
